docs/configuration.md

```diff
@@ -2248,8 +2248,8 @@ should be included on Spark's classpath:
 * `hdfs-site.xml`, which provides default behaviors for the HDFS client.
 * `core-site.xml`, which sets the default filesystem name.
 
-The location of these configuration files varies across CDH and HDP versions, but
-a common location is inside of `/etc/hadoop/conf`. Some tools, such as Cloudera Manager, create
+The location of these configuration files varies across hadoop versions, but
+a common location is inside of `/etc/hadoop/conf`. Some tools create
```

Comment:
This also won't matter either way, but is an example of an instance where naming a specific product is helpful information for users. There's no policy against this. But, it can be removed, sure, it's trivial either way.